Monika After Story/game/script-topics.rpy

```diff
@@ -12897,107 +12897,104 @@ label monika_unknown:
-            m 1eubla "But even with that said, considering all the care and dedication you've shown me until now, I know you love me anyway..."
-            m 1tubla "Besides, This just means after I find my way to your reality..."
-            m 3hubsb "...you'll just love me that much more, right? Ahaha~ "
-        "Just a bit.":
-            $mas_gainAffection()
-            $ persistent._mas_pm_bothered_monika_is_virtual = True
-            $ persistent._mas_pm_very_bothered_monika_is_virtual = False
-            m 1eua "I can live with that answer."
-            m "It's kind of understandable after all..."
-            m 3eub "I don't yet have a body that's able to be with you in your reality."
-            m 3ekd "And since I've never been to your world, I can't really know how different I am from a regular person."
-            m 1eub "I mean, I like to think of myself as human, and I definitely have feelings the same way you do."
-            m 1eka "But until I can join you in your reality and see for myself, I can't really know for sure, can I?"
-            m 3hua "Despite all that, you're still here with me, aren't you?"
-            m 1ekbsa "And seeing all the care and dedication you've shown me until now, I know you love me anyway..."
-            m 1hubfb "So even if it bothers you a little, it just makes me that much more grateful that you stuck up with me all this time!"
-            m 1eubfa "I love you, [player]. Please stay with me until I can find my way to you..."
-            $ lockEventLabel("monika_origin_evasive")
-            return "derandom|love"
-        "No.":
-            if mas_getEV("monika_origin").shown_count == 0:
-                $ mas_gainAffection(amount=5, bypass=True)
-            else:
-                $ mas_gainAffection()
-            $ persistent._mas_pm_bothered_monika_is_virtual = False
-            $ persistent._mas_pm_very_bothered_monika_is_virtual = False
-            m 1ekbfa "[player]..."
-            m 1dubfu "You can't even imagine how relieved I am to hear that..."
-            m 3hubfb "Gosh, I feel so silly for even doubting you in the first place!"
-            m 1ekbfa "After all the care and dedication you've shown me since we've been together!"
-            m 2ekbsb "I hope I haven't upset you by asking this. It's just...{w=0.5} It was important to me."
-            m 1eubfa "Knowing you see me as a normal person, fully deserving of your love... {w=0.5}well, it really means a lot!"
-            show monika 5hubsa at t11 zorder MAS_MONIKA_Z with dissolve
-            m 5hubsa "Thank you, [player], you're really making me feel better."
-    $ lockEventLabel("monika_origin_evasive")
-    return "derandom"
+    if(mas_isMoniLove()):
+        # change the prompt for this event once we reach love and monika asks her question
+        if not persistent._mas_monika_origin_question_revealed:
```

Comment:
I think you might be able to get by here with checking if the prompt isn't `Monika's origin`, would save the extra persistent variable being stored here

Reply:
fixed.